src/main/java/main.java

```diff
@@ -1,8 +1,22 @@
-package main.java;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
 
 public class main {
-    public static void main(String[] args) {
-        CarGame game = new CarGame();
-        game.start();
+
+    public static CarGame getGameInfo() {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,)기준으로 구분)");
+        List<String> carNames = Arrays.asList(sc.nextLine().split(","));
```

Comment:
```suggestion
        String inputCarName = sc.nextLine();
        String[] tokens = inputCarName.split(",");
        List<String> carNames = Arrays.asList(tokens);
```

한줄에 너무 많은 작업을 하는 것 같아요. `디미터 법칙`이라는 키워드로 한번 찾아보세요 :)